.github/dependabot.yml

```diff
@@ -6,6 +6,12 @@ updates:
           interval: 'weekly'
       open-pull-requests-limit: 10
 
+    - package-ecosystem: 'npm'
```

Comment:
@mshima : I wonder if this will work. Because the directory `/generators/client/templates/angular/` contains the file `package.json.ejs`. Unless it's `package.json` I wonder if dependabot will be able to pick up the file and automatically update the versions right? 🤔

Reply:
The versions will be managed by a real package.json https://github.com/mshima/generator-jhipster/blob/master/generators/client/templates/angular/package.json.

With placeholder at package.json.ejs:
https://github.com/mshima/generator-jhipster/blob/d492f1471f1c80f7686c1a1221caffd43c4cb28a/generators/client/templates/angular/package.json.ejs#L160

Dependabot is already triggered:
https://github.com/mshima/generator-jhipster/pull/103

Reply:
@mshima : Ah, I got what you mean now. Great work 😄